src/app/tabs/runTab/contractDropdown.js

```diff
@@ -67,6 +82,11 @@ class ContractDropdownUI {
             ${this.atAddressButtonInput}
           </div>
         </div>
+        <div class="mt-2">
+          ${ipfsCheckbox}
+          <label for="deployAndRunPublishToIPFS" class="text-dark p-0 m-0">PUBLISH TO IPFS</label>
+          <i class="fas fa-info ml-2" aria-hidden="true" title="Publishing to IPFS is allowing to verify the contract code. The bytecode is already stored transparently in the blockchain. Publishing the source code and its ABI will greatly foster contract adoption (auditing, debugging, calling it, etc...)"></i>
```

Comment:
`The bytecode is already stored transparently in the blockchain`
that should be more explanatory:
`This will not compromise your code or making it unsafer (the bytecode is anyway already stored transparently in the blockchain). On the contrary, publishing the source code and its ABI will greatly foster adoption on your contract (auditing, debugging, calling it, etc...)`

Reply:
this is too long for tooltip. if you want to add all these we should think of something else

Reply:
@yann300

Reply:
yeah right that is too much, ok let's keep the short version

Reply:
What is the short version?

Reply:
`Publishing to IPFS facilitate source code verification. Publishing the source code and its ABI will greatly foster contract adoption (auditing, debugging, calling it, etc...)`
is that ok @ioedeveloper @LianaHus

Reply:
A little change to it IMHO.
`Publishing the source code and ABI to IPFS facilitates source code verification and greatly foster contract adoption (auditing, debugging, calling it, etc...)`
@yann300 @LianaHus

Reply:
sounds good

Reply:
Great! I will make the PR.